### PR TITLE
{bio}[intel/2018b] atropos v1.1.21 w/ Python 3.6.6 (+ deps)

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~~#7576~~ (pytest)

### CODE REVIEW

File: easybuild/easyconfigs/a/atropos/atropos-1.1.21-intel-2018b-Python-3.6.6.eb [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+toolchain = {'name': 'intel', 'version': '2018b'}
+
+dependencies = [
+    ('Python', '3.6.6'),
+    ('Pysam', '0.15.1', versionsuffix),
+    ('pytest', '3.8.2', versionsuffix),
+    ('NGS', '2.9.3', versionsuffix),  # required for srastream
+]
+
+#use_pip = True
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/a/atropos/atropos-1.1.21-intel-2018b-Python-3.6.6.eb]: why not?

Comment (boegel) [reply easybuild/easyconfigs/a/atropos/atropos-1.1.21-intel-2018b-Python-3.6.6.eb]: Ah, overlooked, thanks...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3104.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/9a4894648eda7a296bae379bac0ad7ea for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2546.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/c2d5518dc52481023cba8d83836488f5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2111.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/953a09d5eb6745b79d144bbc3a8ac39c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3104.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/63d246db063cdab5b4f58e18b8cc2d41 for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
pg-interactive.hpc.rug.nl - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/0e1734908ae58d3159938025ec61370b for a full test report.

**Review by bedroge — APPROVED** [review]
lgtm

**Comment by bedroge** [comment]
Going in, thanks @boegel!

**Pull request merged**